src/ImageSharp/Formats/Webp/Lossy/LossyUtils.cs

```diff
@@ -971,18 +971,18 @@ private static void DoFilter6(Span<byte> p, int offset, int step)
             int q0 = p[offset];
             int q1 = p[offset + step];
             int q2 = p[offset + step2];
-            int a = WebpLookupTables.Sclip1[(3 * (q0 - p0)) + WebpLookupTables.Sclip1[p1 - q1]];
+            int a = WebpLookupTables.Sclip1[(3 * (q0 - p0)) + WebpLookupTables.Sclip1[p1 - q1 + 1020] + 1020];
 
             // a is in [-128,127], a1 in [-27,27], a2 in [-18,18] and a3 in [-9,9]
             int a1 = ((27 * a) + 63) >> 7;  // eq. to ((3 * a + 7) * 9) >> 7
             int a2 = ((18 * a) + 63) >> 7;  // eq. to ((2 * a + 7) * 9) >> 7
             int a3 = ((9 * a) + 63) >> 7;  // eq. to ((1 * a + 7) * 9) >> 7
-            p[offset - step3] = WebpLookupTables.Clip1[p2 + a3];
-            p[offset - step2] = WebpLookupTables.Clip1[p1 + a2];
-            p[offsetMinusStep] = WebpLookupTables.Clip1[p0 + a1];
-            p[offset] = WebpLookupTables.Clip1[q0 - a1];
-            p[offset + step] = WebpLookupTables.Clip1[q1 - a2];
-            p[offset + step2] = WebpLookupTables.Clip1[q2 - a3];
+            p[offset - step3] = WebpLookupTables.Clip1[p2 + a3 + 255];
+            p[offset - step2] = WebpLookupTables.Clip1[p1 + a2 + 255];
+            p[offsetMinusStep] = WebpLookupTables.Clip1[p0 + a1 + 255];
+            p[offset] = WebpLookupTables.Clip1[q0 - a1 + 255];
+            p[offset + step] = WebpLookupTables.Clip1[q1 - a2 + 255];
+            p[offset + step2] = WebpLookupTables.Clip1[q2 - a3 + 255];
```

Comment:
Bit late to the party but:

Do we always shift with the same value for the same lookup? Shouldn't we define a helper method then or at least add an explanatory comment?

Reply:
> Bit late to the party but

Maybe I was a bit to quick in merging it.

> Do we always shift with the same value for the same lookup?

yes, in C it is defined like this:

```
const uint8_t* const VP8kclip1 = &clip1[255];
``` 
See `dec_clip_tables.c` in the libwebp source.
I thought there might be a way to do the same in C#, but i could not come up with a better solution then this.

Reply:
I will create a follow up PR with helper methods

Reply:
Maybe you could use a `Span` that starts at the `255` offset?